### PR TITLE
Argocd examples

### PR DESCRIPTION
**Pull request by andres-de-castro**
Simple argo cd example to allow deploying of kube-prometheus with the new external library functionality in argocd.

### CODE REVIEW

**Review by paulfantom — APPROVED** [review]
lgtm

File: examples/continuous-delivery/argocd/kube-prometheus/argocd-basic.jsonnet [hunk 1]
```diff
@@ -0,0 +1,13 @@
+local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
+           (import 'kube-prometheus/kube-prometheus-eks.libsonnet') + {
```
Comment (brancz) [inline examples/continuous-delivery/argocd/kube-prometheus/argocd-basic.jsonnet:2]: as a basic example I would prefer to skip this eks patch

**Review by brancz — APPROVED** [review]
Amazing! Just a small change then this lgtm.

File: examples/continuous-delivery/argocd/kube-prometheus/argocd-basic.jsonnet (hunk 1 above)
Comment (stradigi-andres-de-castro) [reply examples/continuous-delivery/argocd/kube-prometheus/argocd-basic.jsonnet]: Done.

**Comment by brancz** [comment]
Very cool! I trust that you tried this out, but I'm also keen on trying this myself! :)

**Pull request merged**